app/views/shf_applications/_shf_application_fields.html.haml

```diff
@@ -42,7 +42,22 @@
                 %tbody.files{'data-toggle': "modal-gallery", 'data-target': "#modal-gallery"}
           .col-md-6
             = render 'uploaded_files_list', shf_application: @shf_application
-
+    %br
+    %p
+      %span.file-delivery-selection
```

Comment:
This section needs to come _before_`.app-section#supporting-files`
The logical UX flow is the a user will tell you how they are going to provide the files.  _Then_ if they are going to upload them now, they will do so.
this also implies that the `.app-section#supporting-files` should be disabled unless the "upload now" button is selected.

Will need to change [feature] tests to verify that.

Reply:
After seeing it in action, my _opinion_ is to move it.  What is more important is what the users think.  So no need to change; let's keep it as you have it, for the reasons you stated.  If users don't respond to it well, we'll change it later. (And, if they do respond well, I will have learned something)